policyengine_us/variables/gov/states/ar/tax/income/ar_non_refundable_credits.py

```diff
@@ -9,4 +9,12 @@ class ar_non_refundable_credits(Variable):
     definition_period = YEAR
     defined_for = StateCode.AR
 
-    adds = "gov.states.ar.tax.income.credits.non_refundable"
+    # adds = "gov.states.ar.tax.income.credits.non_refundable"
+    adds = [
```

Comment:
delete the new code and keep:

    adds = "gov.states.ar.tax.income.credits.non_refundable"